calcom/cal.com
# fix: Function sendScheduledSeatsEmails ignore rule 'Disable default confirmation emails for attendees'

## What does this PR do?

This PR addresses an issue where confirmation emails were incorrectly sent to all attendees, even when the "Disable default confirmation emails for attendees" option was enabled for an event. The problem specifically affected the first attendee, as the function `sendScheduledSeatsEmails` did not have the necessary props (`hostEmailDisabled` and `attendeeEmailDisabled`) to enforce this rule.

Fixes #11677 

## Requirement/Documentation
.
## Type of change




## How should this be tested?

1. Create a Event that offers seats (Tab Advanced - Offer Seats)
2. Create and apply a custom workflow that sends email to the attendee after a event is booked
3. Go back to the event, Advanced Tab and check the option Disable default confirmation emails for attendees
4. Register for the event from different accounts, you will receive only one e-mail, which will be from the workflow.

## Mandatory Tasks
